hlslkit/compile_shaders.py

```
@@ -172,6 +172,7 @@
     strip_debug_defines: bool = False,
     optimization_level: str = "1",
     force_partial_precision: bool = False,
+    debug_defines: set = None,
```

Ruff (0.11.9)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)